package.json

```diff
@@ -90,6 +94,9 @@
   },
   "dependencies": {
     "js-sha3": "^0.8.0",
-    "web3": "^1.3.5"
+    "node-fetch": "^2.6.1",
```

Comment:
Do we use this anywhere?

Reply:
oh we did, but I don't think we do anymore. Will remove